docs/en/deploy/Running-Alluxio-On-Kubernetes.md

```diff
@@ -1689,4 +1689,137 @@ If this is the case, set the following properties to limit the path length:
 > Note: You may see performance degradation due to lack of node locality.
 
   {% endcollapsible %}
+  {% collapsible Worker Pods get OOMKilled by the Kubernetes scheduler %}
+This is most likely caused due to the Kubernetes configured
+[Pod resource limits](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#requests-and-limits)
+having the `limits.memory` set too low.
+
+Firstly, double check the configured values for your Alluxio worker Pod `limits.memory`.
```

Comment:
Note that there's a `LimitRange` thing in K8s that adds default limit/request to all your pods. Maybe in some production clusters, that is configured. So I think it makes sense to check both here and `kubectl describe pod`, and the latter is the source of truth. Not sure how you rephrase to emphasize that point.

Reply:
Thanks, I'll mention that as well!